### ISSUE TITLE
19CVPR| Revisiting Self-Supervised Visual Representation Learning

### ISSUE THREAD

**Issue by XFeiF** [XFeiF/Video_PaperNotes#8]
[paper](https://arxiv.org/abs/1901.09005) && [code](https://github.com/google/revisiting-self-supervised)  

The authors revisit numerous previously proposed self-supervised models, conduct a thorough large scale study and, as a result, uncover multiple crucial in-sights. They challenge a number of common practices in self-supervised visual representation learning and observe that standard recipes for CNN design do not always translate to self-supervised representation learning.

**Comment by XFeiF** [comment]
Previous self-supervised methods focus on the novelty and performance over the benchmark.  
In this paper, the authors explore the upper bound of these self-supervised methods thorough large scale study.  They investigate these methods from the previously unexplored angles.  

And they **uncover multiple important insights**:  
- lessons from the architecture design in the fully-supervised setting do not necessarily translate to the self-supervised setting;
- contrary to previously popular architectures like AlexNet, in residual architectures, the final pre-logits layer consistently results in the best performance;  
- the widening factor of CNNs has a drastic effect on performance of self-supervised techniques;
- SGD training of linear logistic regression may require very long time to converge.

**Comment by XFeiF** [comment]
Another interesting finding:  
**Neither is the ranking of architectures consistent across different methods, nor is the ranking of methods consistent across architectures.**     

This implies that **pretext tasks for self-supervised learning should** not be considered in isolation, but **in conjunction with underlying architectures**.
